TooltipDecorator/Tooltip.module.less

```diff
@@ -83,7 +87,7 @@
 		// Positioning for the arrow and label
 		&.below {
 			.tooltipArrow {
-				top: (@agate-tooltip-offset + 3px);
+				top: (@agate-tooltip-offset + 4px);
```

Comment:
Since 4 is not a multiple of 3, it is better to keep it as the previous value.
Below is same